### ISSUE TITLE
Celestial bodies names are not visible

### ISSUE THREAD

**Issue by daman7cz** [wwwescape/sky-tonight-card#1]
Hello, I have Sky Tonight Native Card.

- I have downloaded the images and config files into path /homeassistant/www/sky-tonight-native-card/
- I have put the lines int configuration.yaml (even though it seems to me it worked even without having it there)
- I have added the card into lovelace.

I see pictures, but the body names are missing. Do you know where the issue can be?

![sky](https://github.com/wwwescape/sky-tonight-card/assets/84605450/dc568000-a97d-430c-a92b-01de1d1e1390)

**Comment by wwwescape** [comment]
Yes, this is an issue with my CSS. I am currently refactoring the code and will fix this as well.
